lib/order.rb

```diff
@@ -0,0 +1,66 @@
+require 'csv'
+require_relative 'customer'
+
+class Order
+  attr_reader :id, :products, :customer, :fulfillment_status
+
+  def initialize(id, product_hash, customer, fulfillment_status = :pending)
+    @id = id
+    @products = product_hash
+    @customer = customer
+    @fulfillment_status = fulfillment_status
+
+    valid_statuses = %i[pending paid processing shipped complete]
```

Comment:
Nice! If you're making it a local variable, maybe it would be a good constant brought out to the class level too?